run.py

```diff
@@ -174,8 +174,8 @@ def main(*args):
 
     args = parser.parse_args()
 
-    if args.bibcodes:
-        args.bibcodes = [x.strip() for x in args.bibcodes.split(',')]
+    #if args.bibcodes:
```

Comment:
what's the problem here? why removed?

Reply:
It was causing errors. I think Steve was next to me and he told me to remove it.

Reply:
regarding page_count, Alberto picked that name to be consistent with all other counts we have.

Reply:
Yes, the comment in commit also says
run.py — commented out two lines, as per Steve instruction.

Reply:
i'm afraid "was causing some errors" is bit too vague to be an acceptable reason for removal

Reply:
Steve should be able to explain the issue, but this is the error
Sorry, cant find the proj home; returning the location of the caller: /home/gshapuri/code/ADSimportpipeline/python/lib/python2.7/site-packages/adsputils

Reply:
Steve should be able to explain this in details. But I just took the comment out and I get this error

Sorry, cant find the proj home; returning the location of the caller: /home/gshapuri/code/ADSimportpipeline/python/lib/python2.7/site-packages/adsputils

and then it hangs. checked the /proj directory and it is accessible.

Reply:
The commenting out was a quick fix.  I'll add a comment with a better suggestion but might not figure that out until after lunch and afternoon meeting.

Reply:
I think we can just use: 
if args.bibcodes:
        args.bibcodes = [x.strip() for x in args.bibcodes]
Since the default value for args.parser is an empty array, the user's input from the command line is already being split on the comma character and an array of strings is automatically initialized.  So, the python list comprehension should not have the split.

Reply:
i see, the argparser actually does the strip() so the whole thing is unnecessary; but we should change the documentation - 'comma separated is not possible in this case, bibcodes have to be space separated'

@golnazads can you please make this change and repush?

Reply:
argparser seems to accept either comma or space separated list and converts to an array of strings.